metadata-schema/closureSchemaClosed.schema.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$id": "/schema/closed-closure",
+  "type": "object",
+  "allOf": [
+    {
+      "if": {
+        "properties": {
+          "closure_type": {
+            "enum": ["Closed", "closed", "CLOSED"]
+          }
+        }
+      },
+      "then": {
+        "properties": {
+          "closure_start_date": {
+            "type": "string",
+            "format": "date"
+          },
+          "closure_period": {
+            "type": "integer",
+            "minimum": 1
+          },
+          "foi_exemption_code": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "type": "string",
+              "$ref": "classpath:/metadata-schema/definitionsSchema.schema.json#/definitions/foi_codes"
+            }
+          },
+          "foi_exemption_asserted": {
+            "type": "string",
+            "format": "date"
+          },
+          "title_closed": {
+            "type": "boolean"
+          },
+          "description_closed": {
+            "type": "boolean"
+          }
+        },
+        "required": [
+          "closure_type",
+          "closure_start_date",
+          "closure_period",
+          "foi_exemption_code",
+          "foi_exemption_asserted",
+          "title_closed",
+          "description_closed"
+        ],
+        "allOf": [
+          {
+            "if": {
+              "properties": {
+                "title_closed": { "const": true }
+              },
+              "required": ["title_closed"]
+            },
+            "then": {
+              "required": ["title_alternate"],
+              "properties": {
+                "title_alternate": {
+                  "type": "string"
+                }
+              }
+            }
+          },
+          {
+            "if": {
+              "properties": {
+                "description_closed": { "const": true }
+              },
+              "required": ["description_closed"]
+            },
+            "then": {
+              "required": ["description_alternate"],
```

Comment:
required handled in required schema (when written)

Reply:
Unless I got the syntax wrong or I'm missing something. This is required for checking:
```
Validate that IF description_closed is Yes THEN
description_alternate is not null (type = string)
```
same with title_alternative

Reply:
I have never tried nested allof if allof but looks good